.npmignore

```diff
@@ -1,36 +0,0 @@
-**/node_modules/
-**/*.log
-test/repo-tests*
-
-# Logs
-logs
-*.log
-
-coverage
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-.nyc_output
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# node-waf configuration
-.lock-wscript
-
-build
-
-# Dependency directory
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
-node_modules
-
-test
-examples
```

Comment:
This file is needed so that the `dist` directory is published to npm - it also prevents the `test` and `examples` directories from being published.

Reply:
We have been removing this file on others repos in favor of the files property in package.json

Reply:
Got it, thanks - did not realise ["files" in package.json](https://docs.npmjs.com/files/package.json#files) did that.